packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/UNSTABLE_EmptyState.twig

```diff
@@ -0,0 +1,83 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _buttonPrimaryText = props.buttonPrimaryText | default(null) -%}
+{%- set _buttonPrimaryUrl = props.buttonPrimaryUrl | default(null) -%}
+{%- set _buttonSecondaryText = props.buttonSecondaryText | default(null) -%}
+{%- set _buttonSecondaryUrl = props.buttonSecondaryUrl | default(null) -%}
+{%- set _description = props.description -%}
+{%- set _headingElementType = props.headingElementType | default('h2') -%}
+{%- set _iconName = props.iconName | default(null) -%}
+{%- set _linkText = props.linkText | default(null) -%}
+{%- set _linkUrl = props.linkUrl | default(null) -%}
+{%- set _title = props.title -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'UNSTABLE_EmptyState' -%}
+{%- set _headingClassName = _spiritClassPrefix ~ 'UNSTABLE_EmptyState__heading' -%}
+{%- set _contentClassName = _spiritClassPrefix ~ 'UNSTABLE_EmptyState__content' -%}
+{%- set _buttonsClassName = _spiritClassPrefix ~ 'UNSTABLE_EmptyState__buttons' -%}
+{%- set _wrapperClassName = _spiritClassPrefix ~ 'UNSTABLE_EmptyState__wrapper' -%}
+{%- set _iconWrapperClassName = _spiritClassPrefix ~ 'UNSTABLE_EmptyState__icon' -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _classNames = [ _rootClassName, _styleProps.className ] -%}
+
+<div
+    {{ mainProps(props) }}
+    {{ styleProp(_styleProps) }}
+    {{ classProp(_classNames) }}
+>
+    <div class="{{ _wrapperClassName }}">
+        {% if _iconName %}
+            <div class="{{ _iconWrapperClassName }}">
+                <Icon name="{{ _iconName }}"/>
+            </div>
```

Comment:
This looks like a Button…

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/0cbe8e50-a7e2-4c7f-9499-364321ced66b)

Reply:
I think we should let this be now. It will be removed later.